include/boost/rational.hpp

```diff
@@ -146,6 +146,9 @@ class rational
     struct helper { IntType parts[2]; };
     typedef IntType (helper::* bool_type)[2];
 
+    class already_normalized {};
```

Comment:
Should this be a public constructor?

Reply:
I think we can leave it this way for now (private).

Reply:
It turns out this is no longer necessary with the removal of reciprocal().